js/interface.js

```diff
@@ -169,8 +183,11 @@ function addImage(image) {
   $imgElement[0].onmouseover = function() {
     $deleteBtn = $(this).find('.image-overlay i');
     $deleteBtn[0].onclick = function() {
+      var imageToDelete = getSelectedImage($imgElement);
+
       onEditClose();
-      data.images.splice(getSelectedImage($imgElement).index, 1);
+      data.images.splice(imageToDelete.index, 1);
+      data.imagesMetadata.splice(imageToDelete.index, 1);
```

Comment:
will these always be in sync or is there a chance they aren't? if it's the latter case I suggest splicing the `imagesMetadata` by using `_.remove` with a specific match

Reply:
If the code will not be changed in the future they should be in sync.